docs/source/adjoint.rst

```diff
@@ -289,11 +289,11 @@ are to be found on `the pygraphviz website
 
 .. _fig-tape:
 
-.. figure:: images/tape.pdf
```

Comment:
Does this mean `tape.pdf` should be removed? Does it exist?

Reply:
:shrug: 
Ideally these things should be being generated dynamically, but we need this for the tutorial this week and the .pdf only renders in Safari, but not Chrome and Firefox.

Reply:
I messed up, that should be an SVG anyway

Reply:
Can you also remove the pdf? Just checked and it isn't used anywhere else so it shouldn't be left lying around.